src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let styles = sourceString.split(';').map(string => string.trim())
+    .filter(string => string !== '');
+
+  styles = styles.map(style => style.split(':')
```

Comment:
Why do you need to rewrite this let?
You can create separate const for each value.
You should use const variables when it is possible.
```suggestion
  const styles = sourceString.split(';').map(string => string.trim())
    .filter(string => string !== '');

  const splitedStyle = styles.map(style => style.split(':')
```
for example.